src/style-auro-calendar.scss

```diff
@@ -0,0 +1,186 @@
+
+/* stylelint-disable color-function-notation, at-rule-empty-line-before */
+
+$mobile-header-height: 86px;
+$mobile-footer-height: 200px;
+
+:host {
+  @include auro_breakpoint($max: $auro-breakpoint-sm) {
+    flex-direction: column;
+  }
+}
+
+.calendarNavBtn {
+  display: flex;
+  width: var(--auro-size-500);
+  height: var(--auro-size-500);
+  align-items: center;
+  justify-content: center;
+  border-width: 1px;
+  border-style: solid;
+  border-color: transparent;
+  border-radius: 50%;
+  background-color: transparent;
+  color: var(--auro-color-border-active-on-light);
+  cursor: pointer;
+
+  &:hover {
+    border-color: var(--auro-color-border-active-on-light);
+  }
+
+  auro-icon {
+    height: var(--auro-size-300);
+    color: var(--auro-color-ui-default-on-light);
+  }
+}
+
+.prevMonth,
+.nextMonth {
+  position: absolute;
+  top: var(--auro-size-200);
+}
+
+.prevMonth {
+  left: var(--auro-size-50);
+}
+
+.nextMonth {
+  right: var(--auro-size-50);
+}
+
+.headerActions {
+  padding: 0 var(--auro-size-200);
+}
+
+.mobileHeader {
+  position: absolute;
+  top: 0;
+  left: 0;
+  display: none;
+  width: 100%;
+  height: $mobile-header-height;
+  flex-direction: row;
+  align-items: center;
+  background-color: var(--auro-color-background-lightest);
+  box-shadow: 0 0 10px rgba(0, 0, 0, .15);
+}
+
+.headerDateFrom {
+  display: flex;
+  height: $mobile-header-height;
+  flex: 1;
+  flex-direction: column;
+  justify-content: center;
+  padding: 0 var(--auro-size-100) 0 var(--auro-size-200);
+  border-left: solid 1px var(--auro-color-border-divider-on-light);
+}
+
+.mobileDateLabel {
+  color: var(--auro-color-text-secondary-on-light);
+  font-size: var(--auro-text-body-size-xs);
+}
+
+.headerDateTo {
+  height: calc($mobile-header-height - var(--auro-size-300));
+  padding: var(--auro-size-300) var(--auro-size-100) 0 var(--auro-size-200);
+}
+
+:host(:not([noRange])) {
+  .headerDateTo {
+    position: relative;
+    display: flex;
+    flex: 1;
+    flex-direction: column;
+    justify-content: center;
+
+    &:after {
+      position: absolute;
+      top: calc(50% + 10px);
+      left: 0;
+      display: block;
+      width: 1px;
+      height: 24px;
+      background-color: var(--auro-color-border-divider-on-light);
+      content: '';
+      transform: translateY(-50%);
+    }
+  }
+}
+
+.mobileFooter {
+  position: absolute;
+  bottom: 0;
+  left: 50%;
+  display: none;
+  width: 100%;
+  height: $mobile-footer-height;
+  flex-direction: column;
+  align-items: flex-end;
+  justify-content: flex-end;
+  padding-bottom: var(--auro-size-200);
+  background: linear-gradient(180deg, rgba(255, 255, 255, 0) 0%, var(--auro-color-background-lightest) 51.56%);
+  pointer-events: none;
+  transform: translateX(-50%);
+}
+
+.mobileFooterActions {
+  position: absolute;
+  bottom: 0;
+  left: 50%;
+  display: none;
+  width: 336px; // match the calendar month width; This needs to be deal with when doing the tiny mobile device layout.
```

Comment:
We should make this a variable at the top of the document. Easier to address if needed to update and reuse if needed.